cellpack/autopack/loaders/migrate_v2_to_v2_2.py

```diff
@@ -0,0 +1,17 @@
+import copy
+
+from ..interface_objects.gradient_data import GradientData, ModeOptions
+
+
+def convert(recipe_data_2_1):
+    new_recipe = copy.deepcopy(recipe_data_2_1)
+    new_recipe["format_version"] = "2.2"
+    for object_name in recipe_data_2_1["objects"]:
+        object_data = recipe_data_2_1["objects"][object_name]
```

Comment:
```suggestion
    for object_name in new_recipe["objects"]:
        object_data = new_recipe["objects"][object_name]
```

Reply:
Tested and working! Modify `object_data` to the deepcopied data instead of the original and then uncomment the `packing_mode` assertion.